docs/settings/storagenetwork/manual.md

```diff
@@ -0,0 +1,311 @@
+---
+sidebar_position: 1
+sidebar_label: Manual Storage Network Configuration
+title: ""
+---
+
+# Manual Storage Network Configuration 
+
+This page describe the maunal method to configure Longhorn Storage Network in Harvester. 
+
+## Prerequisite
+
+1. Make sure that Harvester version is v1.1.0 or above.
+2. Prepare phyical interfaces networks for the storage network, and make sure the VLAN configuration in the external switch side.
+3. Create Network Config in Web UI with the phyical interfaces that you expected for the storage network, and remember the `Cluster Network` field for the next step.
+4. Prepare a IPv4 subnet range in the CIDR format. IP Range should be IPv4 CIDR format and 4 times the number of your cluster nodes. This IP range should not overlap with Kubernetes Cluster IPs and Pods IPs.
+
+### Example Configuration
+
+In the following configuration steps, we will use below configurations for example.
+
+Network Config:
+- ClusterNetwork: test
+- NICs
+  - eth2
+  - eth3
+
+VLANs:
+- Namespace: default
+- Name: storagenetwork-1
+- VLAN ID: 100
+
+IP Range:
+- 192.168.0.0/24
+
+## Configuration
+
+### Step 1: Create VLANs
+
+- Create VLANs in Web UI with desired namespace, name, VLAN ID, Cluster Network, and keep the namespace and name for the further configurations
```

Comment:
I suggest following [SUSE's doc guide](https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-list). A complete sentence in a list should end with a period. Would be nice to check the lists again.

Reply:
fix